libs/gltfio/include/gltfio/ResourceLoader.h

```diff
@@ -52,6 +52,10 @@ struct ResourceConfiguration {
     //! If true, computes the bounding boxes of all \c POSITION attibutes. Well formed glTF files
     //! do not need this, but it is useful for robustness.
     bool recomputeBoundingBoxes;
+
+    //! If true, ignore bind transform for skinned primitives when compute bounding box. Only 
+    //! applicable when recomputeBoundingBoxes is set to true
+    bool ignoreBindTransform;
```

Comment:
@romainguy Do you think adding this config flag is acceptable? If it is, I will try to add it to Android and Web binding

Reply:
yes I think it's fine.